src/styles/main.scss

```diff
@@ -1,7 +1,25 @@
-@import "utils";
-@import "fonts";
-@import "typography";
-
-body {
-  background: $c-gray;
-}
+@import "./utils/mixins";
```

Comment:
this import should be moved below after `@import "./utils/variables";` because it can cause problems if depends of some variables